docs/src/user-guide/installation.md

```diff
@@ -38,7 +38,16 @@ SUBCOMMANDS:
     new      Create new fe project
 ```
 
-# Building from source
+## Package managers
+
+Mac users can also get Fe from [Homebrew](https://brew.sh/). The following command installs Fe and exposes it as `fe` without any further configuration necessary:
```

Comment:
Homebrew is available for Linux as well and can even be installed on >= Windows 10 via the available Linux sub system.

Reply:
```suggestion
Fe can also be installed from [Homebrew](https://brew.sh/). Homebrew is available for Mac, Linux and Windows (via [WSL](https://learn.microsoft.com/en-us/windows/wsl/about)). The following command installs Fe and exposes it as `fe` without any further configuration necessary:
```